client/src/components/ManageArtist/ManageTrackRow.tsx

```diff
@@ -96,6 +96,9 @@ const ManageTrackRow: React.FC<{
           overflow: hidden;
           whitespace: nowrap;
           text-overflow: ellipsis;
+          :hover {
```

Comment:
You don't have to do `&:hover` here for it to work? 

```suggestion
          &:hover {
```